examples/ondisk.py

```diff
@@ -118,7 +118,7 @@
 
     >>> with var.get_local_path() as f:
     ...     with xr.open_dataset(f) as ds:
-    ...         print(ds) # doctest: +ELLIPSIS
+    ...         print(ds) # doctest: +SKIP
```

Comment:
that's the point of tests. Here's the relevant section of the log:

```
File "/home/travis/build/ClimateImpactLab/DataFS/examples/ondisk.py", line 154, in examples.ondisk
Failed example:
    # Acquire the file from the archive and print the version
    with var.get_local_path() as f:
        with xr.open_dataset(f) as ds:
            print(ds) # doctest: +ELLIPSIS
Expected:
    <xarray.Dataset>
    Dimensions:   (location: 3, time: 731)
    Coordinates:
      * location  (location) |S2 'IA' 'IN' 'IL'
      * time      (time) datetime64[ns] 2000-01-01 2000-01-02 2000-01-03 ...
    Data variables:
        tmax      (time, location) float64 12.98 3.31 6.779 0.4479 6.373 ...
        tmin      (time, location) float64 -8.037 -1.788 -3.932 -9.341 ...
    Attributes:
        version: version 2
Got:
    <xarray.Dataset>
    Dimensions:   (location: 3, time: 731)
    Coordinates:
      * location  (location) |S2 'IA' 'IN' 'IL'
      * time      (time) datetime64[ns] 2000-01-01 2000-01-02 2000-01-03 ...
    Data variables:
        tmax      (time, location) float64 12.98 3.31 6.779 0.4479 6.373 4.843 ...
        tmin      (time, location) float64 -8.037 -1.788 -3.932 -9.341 -6.558 ...
    Attributes:
        version:  version 2
**********************************************************************
```

It looks like the changes to xarray's print functions slightly altered the print format. The ellipses are just a little too far out. This test should be modified, not skipped!